<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="2" total="5" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="16006" started-at="2017-12-15T19:24:47Z" finished-at="2017-12-15T19:25:03Z">
    <groups>
    </groups>
    <test name="Extent Report" duration-ms="1263" started-at="2017-12-15T19:24:47Z" finished-at="2017-12-15T19:24:48Z">
      <class name="SeleniumTraining.PHP.ExtentReportsClass">
        <test-method status="PASS" signature="startReport()[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137]" name="startReport" is-config="true" duration-ms="271" started-at="2017-12-16T00:54:47Z" finished-at="2017-12-16T00:54:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method status="FAIL" signature="failTest()[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137]" name="failTest" duration-ms="20" started-at="2017-12-16T00:54:48Z" finished-at="2017-12-16T00:54:48Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at SeleniumTraining.PHP.ExtentReportsClass.failTest(ExtentReportsClass.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- failTest -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137]" name="getResult" is-config="true" duration-ms="18" started-at="2017-12-16T00:54:48Z" finished-at="2017-12-16T00:54:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=failTest status=FAILURE method=ExtentReportsClass.failTest()[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="passTest()[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137]" name="passTest" duration-ms="0" started-at="2017-12-16T00:54:48Z" finished-at="2017-12-16T00:54:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passTest -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137]" name="getResult" is-config="true" duration-ms="0" started-at="2017-12-16T00:54:48Z" finished-at="2017-12-16T00:54:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=passTest status=SUCCESS method=ExtentReportsClass.passTest()[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="SKIP" signature="skipTest()[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137]" name="skipTest" duration-ms="4" started-at="2017-12-16T00:54:48Z" finished-at="2017-12-16T00:54:48Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping - This is not ready for testing ]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping - This is not ready for testing 
	at SeleniumTraining.PHP.ExtentReportsClass.skipTest(ExtentReportsClass.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- skipTest -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137]" name="getResult" is-config="true" duration-ms="0" started-at="2017-12-16T00:54:48Z" finished-at="2017-12-16T00:54:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=skipTest status=SKIP method=ExtentReportsClass.skipTest()[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:SeleniumTraining.PHP.ExtentReportsClass@685cb137]" name="endReport" is-config="true" duration-ms="908" started-at="2017-12-16T00:54:48Z" finished-at="2017-12-16T00:54:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- SeleniumTraining.PHP.ExtentReportsClass -->
    </test> <!-- Extent Report -->
    <test name="PHP Collab" duration-ms="16006" started-at="2017-12-15T19:24:47Z" finished-at="2017-12-15T19:25:03Z">
      <class name="SeleniumTraining.PHP.PhPCollabTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@55f3ddb1]" name="beforeTest" is-config="true" duration-ms="11578" started-at="2017-12-16T00:54:47Z" finished-at="2017-12-16T00:54:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@55f3ddb1]" name="beforeMethod" is-config="true" duration-ms="2" started-at="2017-12-16T00:54:59Z" finished-at="2017-12-16T00:54:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void SeleniumTraining.PHP.PhPCollabTest.CreateUser() throws java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="CreateUser()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@55f3ddb1]" name="CreateUser" duration-ms="3213" started-at="2017-12-16T00:54:59Z" finished-at="2017-12-16T00:55:02Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at SeleniumTraining.PHP.PhPCollabTest.CreateUser(PhPCollabTest.java:82)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateUser -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@55f3ddb1]" name="getResult" is-config="true" duration-ms="0" started-at="2017-12-16T00:55:02Z" finished-at="2017-12-16T00:55:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=CreateUser status=FAILURE method=PhPCollabTest.CreateUser()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@55f3ddb1] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="SKIP" signature="CreateProject()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@55f3ddb1]" name="CreateProject" duration-ms="0" started-at="2017-12-16T00:55:02Z" depends-on-methods="SeleniumTraining.PHP.PhPCollabTest.CreateUser" finished-at="2017-12-16T00:55:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateProject -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@55f3ddb1]" name="afterTest" is-config="true" duration-ms="1201" started-at="2017-12-16T00:55:02Z" finished-at="2017-12-16T00:55:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- SeleniumTraining.PHP.PhPCollabTest -->
    </test> <!-- PHP Collab -->
  </suite> <!-- Suite -->
</testng-results>
